components/Cycles/CycleCard.jsx

```diff
@@ -54,37 +54,45 @@ const WorkoutCard = (props) => {
           {
             icon: 'DELETE',
             text: 'Delete Cycle',
-            onPress: () => Alert.alert('Delete Confirmation', `Are you sure you want to delete ${name}?`, [{
-              text: 'Cancel',
-              style: 'cancel',
+            onPress: () => {
+              if (isSelectedCycle) {
+                Alert.alert('Cannot delete the selected cycle, please select a different cycle and try again.');
```

Comment:
not a big deal, but I don't think the "please select a different cycle and try again." is needed.